use-shopping-cart/rollup.config.js

```diff
@@ -22,6 +23,9 @@ const common = {
       exclude: 'node_modules/**/*',
       transforms: ['jsx']
     }),
+    copy({
+      targets: [{ src: './core/index.d.ts', dest: './dist' }]
+    }),
```

Comment:
We need to add a `.d.ts` file for each file in our dist folder.

<img src="https://user-images.githubusercontent.com/19195374/118375533-67178200-b57f-11eb-8b0e-a4df9bae5e5e.png" alt="Tree representation of the dist folder: core and react file's with .es.js, .js, and .umd.js files each with .map files as well." width="400">

We need:

- `core.es.d.ts`
- `core.d.ts`
- `core.umd.d.ts`
- `react.es.d.ts`
- `react.d.ts`
- `react.umd.d.ts`